inst/pages/exercises.qmd

```diff
@@ -1703,17 +1706,27 @@ Create an interactive heatmap.
 #| code-summary: "Show solution"
 #| eval: false
 
-# Load necessary libraries
+# Load required libraries
+library(mia)
 library(heatmaply)
 
-# Create example data
-example_data <- matrix(rnorm(200), nrow = 20, ncol = 10)
+# Load the GlobalPatterns dataset from the mia package
+data("GlobalPatterns", package = "mia")
+
+# Here we extract the first 20 taxa and the first 10 samples as an example
+taxa_data <- assay(GlobalPatterns)[1:20, 1:10]
+taxa_names <- rownames(taxa_data)
+sample_names <- colnames(taxa_data)
```

Comment:
are these used below (I do not immediately find)? If not, then better to remove lines 1718-1719